site/docs/bin.md

```diff
@@ -69,15 +82,19 @@ Setting the `maxbins` parameter changes the number of output bins.
 
 <div class="vl-example">
```

Comment:
This one has more number of bin.   

(I use `histogram`  right above here if you see the full file.)